discord_extensions/generic.py

```diff
@@ -14,6 +14,10 @@ async def help(self, ctx):
             description="``< >`` = Required arguments\n``[ ]`` = Optional arguments",
             colour=0x1ABC9C, timestamp=ctx.message.created_at
         )
+        embed.add_field(
+            name="Command Prefix",
+            value=f"Prefix: ``{DiscordConfig.prefix}``"
+        )
```

Comment:
Not really needed, it's on all the commands already

Reply:
For idiots who can't see. Believe, I have had to deal to such ppl. So, I added this for them.